core/contracts/contracts/privacy/IShield.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: CC0
+pragma solidity ^0.6.9;
+
+interface IShield {
+
+    function getVerifier() external view returns (address);
+    function verifyAndPush(
```

Comment:
Let's name these params here in the interface for readability.

Reply:
Done